api/src/ol/ol.module.ts

```diff
@@ -65,6 +66,16 @@ for (const role of roles) {
   }
 }
 
+// Centralize queue definitions in an array for better reusability
+const queues = [
+  { name: "ol-clickhouse-ingestor", connection: redisClient},
```

Comment:
you can simplify this even further by listing only the queue names in a `string[]`.